core/profiles/standard/standard.install

```diff
@@ -106,6 +106,41 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
+  // Enables the title_combo block and positions breadcrumbs
+  // immediately above title_combo block.
+
+  $title_combo_uuid = 'a549d0bc-51f7-46b0-9b41-653788410dd4';
```

Comment:
Perhaps we should be generating a random UUID here, instead of hard-coding it:

```suggestion
  $uuid = new Uuid();
  $title_combo_uuid = $uuid->generate();
```

Reply:
Maybe. I didn't think so, because we are not generating random uuids in our current core layouts. They are hard-coded. See:

core/modules/layout/config/layout.layout.default.json
core/modules/layout/config/layout.layout.admin_default.json

Is this a security issue or simply a tool to prevent conflicting ids? If this is just about avoiding conflicts, then I don't see that a new random id is necessary in every case.

Reply:
I think I would defer to a core committer regarding this change. I don't object to this change. It's not clear to me if this is helpful or not.